miden-lib/src/transaction/procedures.rs

```diff
@@ -0,0 +1,239 @@
+            5994228928952574041,
+            3477745056362616903,
+            1514247258411024664
+        ),
+        // burn_asset
+        digest!(
+            2613729911428583836,
+            3409713366391106845,
+            4618787175150657117,
+            13550289764852635265
+        ),
+        // get_fungible_faucet_total_issuance
+        digest!(
+            5567752045855424912,
+            1313115426050254227,
+            12797601829399057688,
+            10963909072124913328
+        ),
+        // mint_asset
+        digest!(
+            11833462197234963951,
+            521365474344899632,
+            12219339115593432087,
+            3026752009521887157
+        ),
+        // add_asset_to_note
+        digest!(
+            14950372948135486500,
+            16795936443880575519,
+            10590371208545379138,
+            3493796503081669802
+        ),
+        // create_note
+        digest!(
+            14778603067944873506,
+            14071319835769664212,
+            13946705703761691189,
+            68248535266635199
+        ),
+        // get_input_notes_commitment
+        digest!(
+            2019728671844693749,
+            18222437788741437389,
+            12821100448410084889,
+            17418670035031233675
+        ),
+        // get_note_assets_info
+        digest!(
+            12346411220238036656,
+            18027533406091104744,
+            14723639276543495147,
+            11542458885879781389
+        ),
+        // get_note_inputs_hash
+        digest!(
+            17186028199923932877,
+            2563818256742276816,
+            8351223767950877211,
+            11379249881600223287
+        ),
+        // get_note_sender
+        digest!(
+            15233821980580537524,
+            8874650687593596380,
+            14910554371357890324,
+            11945045801206913876
+        ),
+        // get_note_serial_number
+        digest!(
+            203467101694736292,
+            1871816977533069235,
+            11026610821411620572,
+            8345006103126977916
+        ),
+        // get_output_notes_hash
+        digest!(
+            4412523757021344747,
+            8883378993868597671,
+            16885133168375194469,
+            15472424727696440458
+        ),
+        // get_block_hash
+        digest!(
+            15575368355470837910,
+            13483490255982391120,
+            5407999307430887046,
+            13895912493177462699
+        ),
+        // get_block_number
+        digest!(
+            957081505105679725,
+            18012382143736246386,
+            13337406348155951825,
+            4537613255382865554
+        ),
+    ];
+
+    /// Array of all available kernels.
+    pub const PROCEDURES: [&'static [Digest]; 1] = [&Self::KERNEL0_PROCEDURES];
+
+    /// Returns procedures of the kernel specified by the `kernel_offset` as vector of Felts.
+    pub fn procedures_as_elements(kernel_offset: u8) -> Vec<Felt> {
```

Comment:
Maybe instead of calling this `kernel_offset` we should call it `kernel_version`? So, for now we'll have kernel version 0, but later on we'll add v1, v2 etc.

Reply:
Yep, I'll rename it (probably at first I slightly misunderstood the idea, so I thought that we will have several different kernels instead of different versions of one kernel)